tests/helpers/start-app.js

```diff
@@ -10,6 +11,7 @@ export default function startApp(attrs) {
 
   Ember.run(function() {
     application = Application.create(attributes);
+    Ember.Test.registerAsyncHelper('runGist', runGist);
```

Comment:
This only needs to happen once, when the module is loaded.  Including it here (inside the `startApp` function) will have this ran once for each acceptance test.

I prefer to have the `run-gist` helper imported and registered in `tests/test-helper.js`.  Any objections?